templates/.github/workflows/ci.yml

```diff
@@ -67,15 +67,15 @@ jobs:
         - uses: actions/checkout@v2
         - name: Set IMAGE_TAG as env
           run: |
-            APP_NAME=$(echo "${{ github.repository }}" | sed -e 's/.*\///g')
+            APP_NAME=<% .Name %>
             IMAGE_TAG=${APP_NAME}-$(git rev-parse --short=7 ${{ github.sha }})
             echo "IMAGE_TAG=${IMAGE_TAG}" >> $GITHUB_ENV
         - name: Set APP_NAME and NAMESPACE as env
           run: |
-            NAMESPACE=$(echo "${{ github.repository }}" | sed -e 's/.*\///g')
-            echo "NAMESPACE=${NAMESPACE}" >> $GITHUB_ENV
-            APP_NAME=$(echo "${{ github.repository }}" | sed -e 's/.*\///g')
+            APP_NAME=<% .Name %>
```

Comment:
Does this need to be duplicated across the various steps, or can we just define it once?

Reply:
you can define this above Line 14, then the rest of the workflow should be able to pick it up ( I think)

Reply:
Cool, that worked. Example build is here https://github.com/commit-app-playground/lucas-backend-test-0720/runs/3115156200?check_suite_focus=true